tests/python/cli/test_cli.py

```diff
@@ -71,15 +71,15 @@ def fxt_backup_file(self, fxt_new_task: Task, fxt_coco_file: str):
 
     @pytest.fixture
     def fxt_new_task(self):
-        files = generate_images(str(self.tmp_path), 5)
+        files = generate_images(self.tmp_path, 5)
 
         task = self.client.tasks.create_from_data(
             spec={
                 "name": "test_task",
                 "labels": [{"name": "car"}, {"name": "person"}],
             },
             resource_type=ResourceType.LOCAL,
-            resources=files,
+            resources=list(map(os.fspath, files)),
```

Comment:
Probably, it would be nice to support these paths in SDK APIs as well.

Reply:
Yeah, I was going to implement that at some point.